apps/pie-docs/src/patterns/cookie-banner/overview/overview.md

```diff
@@ -0,0 +1,247 @@
+width: 120
+} %}
+
+---
+
+## Interaction area
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/interaction-area-banner.svg",
+alt: "Banner with highlights on the areas of interaction: links and buttons.",
+width: 120
+} %}
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/interaction-area-modal.svg",
+alt: "Modal with highlights on the areas of interaction: links and buttons.",
+width: 120
+} %}
+
+{% list {
+    type: listTypes.highlight,
+    highlightColour: ["support-brand-03", "support-brand-04", "support-brand-07", "support-brand-05", "support-brand-06"],
+    items: [
+        "**Redirect:**  It will redirect the user to the page  they have selected in a new tab, the cookie banner would appear until the user makes a selection.",
+        "**Confirm selection:**  It saves the user's selection and closes the cookie banner/modal.",
+        "**Opens a “Manage preferences” modal:**  A modal will open and allow user to manage their preferences. ",
+        "**Toggle on/off:**  Allows the user to toggle selections on or off.",
+        "**Back button:**  It closes the modal and resurfaces the cookie banner."
+    ]
+} %}
+
+---
+
+## Content
+
+{% notification {
+  type: "warning",
+  iconName: "alert-triangle",
+  message: "The ‘header’, ‘body copy’, and ‘button labels’ are set and should not be changed, unless you need to translate them into another language."
+} %}
+
+---
+
+## Behaviours
+
+### Background
+
+#### Banner
+
+The background page of the cookie banner isn’t disabled, and is only disabled once the modal is opened. The “Select your country” popover from the top navigation should sit on top of the banner so that the users can change the locale.
+
+{% contentPageImage {
+  src:"../../../assets/img/patterns/cookie-banner/behaviours-background-banner.svg",
+  alt: "Banner possitioned at the bottom with the “Select your country” popover on top of the banner.",
+  width: 120
+} %}
+
+#### Modal
+
+The background page is disabled when the modal is opened until the user has made a selection and is ready to continue with their flow, or returns to the banner. A scrim is applied and the banner disappears.
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/behaviours-background-modal.svg",
+alt: "Cookie modal that positioned in the middle of the screen, on top of a dark overlay.",
+width: 120
+} %}
+
+#### Turn all on / off
+
+Once the overarching toggle is toggled on/off, the child toggles follow suit. The overarching toggle will not be toggled on unless all child toggles are selected.
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/behaviours-turn-all-on-off.svg",
+alt: "Modal showing the toggles states when \"Toggle All\" is or is not selected.",
+width: 120
+} %}
+
+
+---
+
+## Positioning
+
+### Banner
+
+The cookie banner is always positioned at the bottom of the screen with default external bottom, left and right padding of 16px.
```

Comment:
Margin instead of external padding?

Reply:
I think it refers to the padding of the container of the banner. Adding margin to the cookie banner wouldn't make sense I believe. So Maybe that is what needs to be clarified?

> The cookie banner is always positioned at the bottom of the screen with default bottom, left and right padding of 16px in the container of the banner.